pkg/steampipeconfig/parse/decode_options.go

```diff
@@ -9,37 +9,53 @@ import (
 )
 
 // DecodeOptions decodes an options block
-func DecodeOptions(block *hcl.Block) (options.Options, hcl.Diagnostics) {
+func DecodeOptions(block *hcl.Block, settings ...WithDecodeSetting) (options.Options, hcl.Diagnostics) {
```

Comment:
I though we were renaming WithDecodeSetting to blockMappingOverride or something?